querqy-core/src/main/java/querqy/model/Term.java

```diff
@@ -71,6 +71,10 @@ public ComparableCharSequence getValue() {
       return value;
    }
 
+   public void setValue(ComparableCharSequence value) {
```

Comment:
I think there is no reason to make the Term value mutable and it seems that `setValue()` is never called. Can we revert this?